src/main/frontend/pipeline-console-view/pipeline-console/main/DataTreeView.tsx

```diff
@@ -69,23 +68,37 @@ const getTreeItemsFromStepList = (stepsItems: StepInfo[]) => {
         key={stepItemData.id}
         nodeId={String(stepItemData.id)}
         label={stepItemData.name}
+        stageId={String(stepItemData.stageId)}
```

Comment:
there's an error in the browser console that is likely this line (although could be another)

<img width="758" alt="image" src="https://user-images.githubusercontent.com/21194782/192383832-5f685760-2e2f-449a-b0b7-4dd238a62f38.png">